app/javascript/mastodon/components/video_player.js

```diff
@@ -9,7 +9,6 @@ const messages = defineMessages({
   toggle_sound: { id: 'video_player.toggle_sound', defaultMessage: 'Toggle sound' },
   toggle_visible: { id: 'video_player.toggle_visible', defaultMessage: 'Toggle visibility' },
   expand_video: { id: 'video_player.expand', defaultMessage: 'Expand video' },
-  expand_video: { id: 'video_player.video_error', defaultMessage: 'Video could not be played' }
```

Comment:
It is use at [line 166](https://github.com/tootsuite/mastodon/blob/2a8e034472d8cc761370b9be499138f035114b25/app/javascript/mastodon/components/video_player.js#L166), but this does not use defineMessages.